card/lib/card/migration.rb

```diff
@@ -62,12 +62,10 @@ def contentedly
     Card::Cache.reset_all
     Cardio.schema_mode "" do
       Card::Auth.as_bot do
-        ActiveRecord::Base.transaction do
```

Comment:
no need for a screenshot. I just figured out that github is smart enough to recognize commit SHAs in comments 5e22747